understanding/20/focus-order.html

```diff
@@ -53,25 +53,27 @@ <h2>Intent of Focus Order</h2>
+      <p>Focus order needs to allow the user to navigate focusable elements in a logical order,
+         and that order needs to preserve any meaning or operation that the page is conveying. 
+         Focus order does not necessarily need to follow the visual layout of the web page,
          as long as the order in which elements receive focus is logical, and the hierarchy and relationship
          of content implied by the visual presentation is preserved. For instance, in a two-column
-         layout, the assumption in western left-to-right systems may be that elements in the
-         left-hand column would receive focus first – and, more generally, that focus order proceeds across lines,
-         from components in the top-left corner of the viewport to the bottom-right corner.
-         However, if meaning and operation are not affected, in this case it is not a failure if elements in the right-hand
-         column receive focus first. As a best practice, though, make sure the focus order reinforces the reading order.
+         layout, the assumption in western left-to-right systems may be that focus moves through the elements in the
+         left-hand column first, from the top to the bottom of the column, and then proceed to the right-hand column.
+         However, if the two columns are independent of each other, and meaning/operation are not affected, it is not
+         a failure if elements in the right-hand column receive focus first, followed by the elements in the left-hand column.
+         As a best practice, though, make sure the focus order reinforces the reading order implied by the visual layout.
       </p>
       
       <div class="note">
          <p>It is not a failure of this success criterion if static/non-interactive content
          receives focus. Making an element that does not act as an actionable user interface component
          focusable, or programmatically moving focus to such an element, does not necessarily constitute
-         a failure of this success criterion. This situation is a failure if these additional focus stops
-         impede the operation of the content, or create illogical focus orders - for example, a control
-         appearing to receive focus multiple times due to the use of nested focusable elements.</p>
+         a failure of this Success Criterion. This situation is a failure if these additional focus stops
+         impede the operation of the content, or create confusing and/or illogical focus orders - for example,
+         a control appearing to receive focus multiple times due to the use of nested focusable elements.</p>
          <pre><code>&lt;div tabindex="0"&gt;&lt;button&gt;...&lt;/button&gt;&lt;/div&gt;</code></pre>
-         <p>As a best practice, avoid having focusable elements which cannot be operated
+         <p>As a best practice, avoid having too many focusable elements which cannot be operated
```

Comment:
I wonder if instead of "too many" this could be something like "unwarranted"? 
I think "too many" is overly generous.